terraform/key_vault.tf

```diff
@@ -63,12 +63,6 @@ resource "azurerm_key_vault_secret" "drs-contract-staging" {
   value        = var.drs_contract_staging
   key_vault_id = azurerm_key_vault.hro-scheduling-api-key-vault.id
 }
-
-resource "azurerm_key_vault_secret" "drs-priority-staging" {
```

Comment:
We should also remove drs_priority_production and drs_priority_staging from the Github yml file as they are no longer used.
Also, the github secrets should be removed to prevent confusion.

Reply:
Good catch!

Have removed from [GitHub action definition file](https://github.com/Newarkandsherwood/HousingRepairsSchedulingApi/pull/23/files#diff-c12e736384cad57e328ffb7f2f69487818591020f2e6945b4230f91d406210d9).

Will remove from secrets after merging.